samples/BenchmarkDotNet.Samples.Forms/BenchmarkDotNet.Samples.Forms.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Collections.Immutable" Version="1.7.0" />
+    <PackageReference Include="Xamarin.Forms" Version="4.5.0.530" />
+    <PackageReference Include="Xamarin.Essentials" Version="1.5.2" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\BenchmarkDotNet\BenchmarkDotNet.csproj" />
+    <ProjectReference Include="..\BenchmarkDotNet.Samples\BenchmarkDotNet.Samples.csproj" />
```

Comment:
is the dependency on `BenchmarkDotNet.Samples` necessary here? Could we remove it? If so, could you please also remove it from the `BenchmarkDotNet.Xamarin` solution and revert changes applied to `BenchmarkDotNet.Samples.csproj`?

```suggestion
```

Reply:
I just added a single link to this instead, which should be fine:
```xml
<Compile Include="..\BenchmarkDotNet.Samples\IntroBasic.cs" Link="Benchmarks\IntroBasic.cs" />
```

Reply:
@jonathanpeppers great, thank you!